elasticsearch-curator/actions.yml

```diff
@@ -0,0 +1,3 @@
+curator_actions: []
```

Comment:
Maybe is good idea to add default curator_action configuration, because when this option is empty list docker image  will return error.

Reply:
Done